data/projekte/01-klimadaten.yaml

```diff
@@ -0,0 +1,46 @@
+title: Klimadaten
```

Comment:
```suggestion
title: Klimawandel
```

Reply:
I am sorry to change yet another title, but this way it makes more sense in combination with the other two boxes: two topics of interest, and then the third box an explanation of the basics behind our work. (Klimawandel = a topic; Kommunalpolitik = a topic; Daten - die Grundlage unserer Arbeit = an explanation)